pkg/registry/experimental/controller/etcd/etcd_test.go

```diff
@@ -82,8 +83,11 @@ var validScale = extensions.Scale{
 }
 
 func TestGet(t *testing.T) {
-	storage, server, si := newStorage(t)
-	defer server.Terminate(t)
+	storage, server, si, destroyFunc := newStorage(t)
+	defer func() {
+		destroyFunc()
+		server.Terminate(t)
+	}()
```

Comment:
Just personal coding style.
Usually the less "defer" the more efficient the program is. But this is test anyway.